scripts/demo_content.py

```diff
@@ -0,0 +1,407 @@
+        password=User.hash_password("AyCaramba!")
+    )
+
+    User.create(
+        user_id="859515a2-4d56-4003-a99c-a13ec906e9b1",
+        first_name="Brian",
+        last_name="Smith",
+        email="mrsmith@yahoo.com",
+        password=User.hash_password("ThisIsASecret")
+    )
+
+    User.create(
+        user_id="7343ef59-60e4-47f7-8c6a-90728e53e955",
+        first_name="Pedro",
+        last_name="Perez",
+        email="predito@hotmail.com",
+        password=User.hash_password("ElpapaDeLosHelados")
+    )
+
+    User.create(
+        user_id="b2ec03fd-ab2c-4a88-abed-0f381d0f9e56",
+        first_name="Paolo",
+        last_name="Giordano",
+        email="bucanero2008@yahoo.com",
+        password=User.hash_password("###goodkama***")
+    )
+
+    User.create(
+        user_id="a342521d-8643-4ae1-aedb-59a3a0f44a0c",
+        first_name="Naoki",
+        last_name="Fukuyama",
+        email="isawgozilla@yahoo.co.jp",
+        password=User.hash_password("KawaYama")
+    )
+
+    User.create(
+        user_id="72d02f9f-c3db-49a6-887f-4ee09e2e52d7",
+        first_name="Jacopo",
+        last_name="Mancini",
+        email="foza_viola1972@yahoo.com",
+        password=User.hash_password("OhFiorentina!")
+    )
+
+    User.create(
+        user_id="44ee810a-c925-4fa9-aacd-760ad116f166",
+        first_name="Rolan",
+        last_name="Adamoli",
+        email="inesadamoli@yahoo.co.uk",
+        password=User.hash_password("ksdkhgs78¿")
+    )
+
+    User.create(
+        user_id="91838e23-f95b-41b7-ac49-1951354ecfb1",
+        first_name="Livia",
+        last_name="Nunez",
+        email="livianunez@hotmail.com",
+        password=User.hash_password("lisbonaèbona")
+    )
+
+    User.create(
+        user_id="1e181835-bf05-4557-92d6-6415c3da9a9c",
+        first_name="Neve",
+        last_name="Turchi",
+        email="snow_turchi@hotmail.com",
+        password=User.hash_password("biancaneve_dorme")
+    )
+
+    User.create(
+        user_id="77741629-eb3c-4226-9a0b-94c71617052d",
+        first_name="I'timad",
+        last_name="Cuevas",
+        email="lacaverna2001@gmail.com.com",
+        password=User.hash_password("CosmicCromagnon")
+    )
+
+    User.create(
+        user_id="27efd008-e910-4656-b5aa-0214577f580d",
+        first_name="Ziauddin",
+        last_name="O'Brian",
+        email="bigbrother2017@gmail.com",
+        password=User.hash_password("IgnoranceIsPower")
+    )
+
+    User.create(
+        user_id="01357042-214b-4224-8c6f-2d821fc40b37",
+        first_name="Benedita",
+        last_name="Chavarria",
+        email="beneditach@gmail.com",
+        password=User.hash_password("ElChapulinColorado")
+    )
+
+    User.create(
+        user_id="025a0003-e1be-4866-aa74-81947f44b643",
+        first_name="Alejandra",
+        last_name="Texeira",
+        email="alejandratch@gmail.com",
+        password=User.hash_password("allYouNeedIsLove")
+    )
+
+    # start create items
+
+    Item.create(
```

Comment:
Idem, cerchiamo di generare queste cose. E se usiamo un random, basta settare un seed fissato all'inizio in modo tale che i risultati siano riproducibili.